tests/testit/test-formats.R

```diff
@@ -42,3 +42,4 @@ test_format("peerj_article")
 test_format("amq_article")
 test_format("mdpi_article")
 test_format("mnras_article")
+test_format("gigascience_article")
```

Comment:
That is all you need to do about testing.

Reply:
I want to test it before I submit/merge my PR.  That is why `testthat` would be helpful to me.  As I said, I found a workaround, but it would be convenient not to need to work around this. Thanks.